docs/root/configuration/upstream/health_checkers/thrift.rst

```diff
@@ -15,15 +15,11 @@ check request must be configured to enable Thrift health checks. The sequence id
 An example for :ref:`custom_health_check <envoy_v3_api_msg_config.core.v3.HealthCheck.CustomHealthCheck>`
 using the Thrift health checker is shown below:
 
-
-.. code-block:: yaml
-
-  custom_health_check:
-    name: envoy.health_checkers.thrift
-      typed_config:
-        "@type": type.googleapis.com/envoy.extensions.health_checkers.thrift.v3.Thrift
-        method_name: ping
-        transport: HEADER
-        protocol: BINARY
+.. literalinclude:: _include/thrift-health-check.yaml
+    :language: yaml
+    :lines: 30-39
```

Comment:
It looks like the equivalent of the old code block would be lines 48-54, not 30-39?